tests/tests/test_filetransfer.py

```diff
@@ -629,7 +629,9 @@ def test_filetransfer_limits_download_err_not_a_regular_file(
         r = download_file(path, self.devid, self.auth_token)
 
         assert r.status_code == 400, r.json()
-        assert r.json().get("error").endswith("path is not a regular file")
+        assert (
+            r.json().get("error") == "file transfer failed: path is not a regular file"
+        )
```

Comment:
Do we even need to revert this?

Reply:
Not really. It is a harmless change but also not necessary here as we are not updating the backend. I reverted them for the sake of correctness. The same applies to the other commit.